docs/administration/github-org.rst

```diff
@@ -1,10 +1,103 @@
+###################
 GitHub organization
-===================
+###################
 
 This page explains how the `RITlug GitHub organization`_ is organized and how
-it is intended to be used.
+to manage it.
 
-.. _`RITlug GitHub organization`: https://github.com/RITlug
 
-…more to come soon.
+*******
+Purpose
+*******
+
+The GitHub organization is a central place to manage club projects, membership,
+and infrastructure. Most data is static and doesn't change often. Setting up
+club projects or adding a new member is when you usually need to interact with
+the GitHub organization.
+
+
+************
+Repositories
+************
+
+There are no restrictions on repositories allowed. Repository creation
+permission is granted only to eboard members as a precaution. This approach was
+decided to ensure that the current eboard has a chance to review a project
+before officially endorsing it underneath our GitHub organization.
+
+Generally, repository creation should be encouraged rather than discouraged.
+
+
+******************
+Adding new members
+******************
+
+There is no criteria for adding new members to the RITlug GitHub organization.
+There are no requirements to add new members to the RITlug GitHub organization
+other than being an active community member. In the future, if a membership
+criteria is developed, this section should be updated.
+
+Active community members of RITlug should be added to the GitHub organization.
```

Comment:
With the new commit, this becomes redundant. Maybe a horizontal line if you don't want to keep it like this? I'm going to dismiss my blocking review, but I'm not going to approve the PR since this feels awkward to me.

Reply:
@jflory7 ^

Reply:
I think it's okay. It's important we keep good habits of keeping things well-maintained over time – commit early, commit often. If it's not perfect now, we can always revisit. I think a membership discussion is more important to clarifying this, but it won't be solved relatively soon to block this PR.